website/client/src/components/HeaderBar/HeaderBar.js

```diff
@@ -39,6 +45,7 @@ export default class HeaderBar extends React.Component {
                             <Button className="bttn-login" title="Login" onButtonClick={this.handleLogin} />
                         </div>
                         {this.state.loginClicked ? <LoginModal onClose={this.handleCloseLogin} /> : null}
+                        {this.state.registerClicked ? <RegisterModal onClose={this.handleCloseRegister} /> : null}
```

Comment:
You shouldn't use conditional render for modal(just in special case), Modal has a prop name "isOpen" (I don't remember exactly) for setting opening it.